### ISSUE TITLE
Unable to process command '::set-env name=XXX::yyy' successfully.

### ISSUE THREAD

**Issue by maoosi** [nick-invision/persist-action-data#2]
Github recently deprecated set-env and add-path commands following a security vulnerability (see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/).

Since today, using **nick-invision/persist-action-data@v1** in a workflow returns the following error:

```
Unable to process command '::set-env name=persist-action-data-XXX::yyy' successfully.

The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

@nick-invision From the Github post, it seems like updating the `@actions/core` package to v1.2.6 or greater might be sufficient to get the updated addPath and exportVariable functions, and therefore make the package work?

**Comment by nick-invision** [comment]
Thanks for reporting this @maoosi, I knew about the deprecation of those commands but didn't realize that `exportVariable` called `set-env` behind the scenes.  I have updated the dependency and published both `v1.1` as well as moved the `v1` tag so this should be unblocked.

**Comment by maoosi** [comment]
@nick-invision Awesome, I've just tried again and it's working fine. Thanks for fixing so quickly.

BTW, super useful Action 👍! It's making my life easier on some deploy workflows I have in place.
